optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/PlannerBenchmarkConfig.java

```diff
@@ -320,6 +297,39 @@ protected int resolveParallelBenchmarkCount() {
         return resolvedParallelBenchmarkCount;
     }
 
+    protected int resolveParallelBenchmarkCountAutomatically(int availableProcessorCount) {
+        // TODO Tweak it based on experience
+        if (availableProcessorCount < 2) {
+            return availableProcessorCount;
```

Comment:
This is the only functional change I've made. Although I think it can hardly happen, if the available processor count were 0 or less, the config will no fail on the `resolvedParallelBenchmarkCount < 1` check instead of reducing `resolvedParallelBenchmarkCount` to 0 and logging a warning (see lines 286-296).